charts/console/README.md

```diff
@@ -985,6 +986,44 @@ platform:
 This example will install a `ServiceMonitor` and a `GrafanaDashboard` in the namespace `monitoring-namespace`.
 The `ServiceMonitor` will scrape metrics from Conduktor Console every 30 seconds and the `GrafanaDashboard` will be available in Grafana instance with label `grafana: tooling` in the folder `tools` and use Prometheus datasource named `my-prometheus-ds`.
 
+### Install with custom certificates
+*NOTE:* The example is for a `keystore` but the same technique could be applied if you need to configure an additional `truststore` or `keytab`.
+
+It's recommended to load your certificates as secrets.
+```shell
+kubectl create secret generic conduktor-console-keytab \
+    --from-file=./my.keytab
+```
+
+We can then proceed to mount the secret as a volume.
+Note the `mountPath` which is where our cert will be accessible from.
+```yaml
+platform:
+  extraVolumes:
+    - name: kafka-keytab
+      secret:
+        secretName: conduktor-console-keytab
+  extraVolumesMounts:
+    - name: kafka-keytab
+      readonly: true
+      mountPath: /etc/kafka
+```
+
+Finally, your cluster configuration should look something like this:
+```shell
+sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required useKeyTab=true storeKey=true keyTab="/etc/kafka/my.keytab" principal="admin/for-kafka@TEST.CONDUKTOR.IO";
+sasl.mechanism=GSSAPI
+security.protocol=SASL_SSL
+sasl.kerberos.service.name=kafka
+```
```

Comment:
Maybe explain it in context of chart values like this
```suggestion
```yaml
config: 
  clusters:
    - id: my-cluster
      bootstrapServers: kafka:9092
      properties: |
          sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required useKeyTab=true storeKey=true keyTab="/etc/kafka/my.keytab" principal="admin/for-kafka@TEST.CONDUKTOR.IO";
          sasl.mechanism=GSSAPI
          security.protocol=SASL_SSL
          sasl.kerberos.service.name=kafka
```
```